app/products/calls/alerts.ts

```diff
@@ -23,15 +24,21 @@ let recordingAlertLock = false;
 // Only unlock if/when the user starts a recording.
 let recordingWillBePostedLock = true;
 
-export const showLimitRestrictedAlert = (maxParticipants: number, intl: IntlShape) => {
+export const showLimitRestrictedAlert = (info: LimitRestrictedInfo, intl: IntlShape) => {
     const title = intl.formatMessage({
-        id: 'mobile.calls_participant_limit_title',
-        defaultMessage: 'Participant limit reached',
+        id: 'mobile.calls_participant_limit_title_GA',
+        defaultMessage: 'This call is at capacity',
     });
-    const message = intl.formatMessage({
+    let message = intl.formatMessage({
         id: 'mobile.calls_limit_msg',
         defaultMessage: 'The maximum number of participants per call is {maxParticipants}. Contact your System Admin to increase the limit.',
```

Comment:
Interesting, does this mean we weren't showing anything special for Cloud before? It's not like the sysadmin could have done anything to raise limits.

Reply:
@streamer45 We were showing the generic max participants message, so correct: nothing special for cloud.